src/flp.rs

```diff
@@ -936,6 +940,7 @@ mod tests {
     }
 
     impl<F: FieldElement> Type for Issue254Type<F> {
+        const ID: u32 = 0xFFFF3003;
```

Comment:
No need to make these test codepoints unique. In fact, I think it's better if they're the same so that someone reading this code in the future doesn't think this is important.